LICENSE

```diff
@@ -1,25 +1,29 @@
-Copyright (c) 2013-2018, OVH SAS.
+BSD 3-Clause License
```

Comment:
This file has been updated and fixed on the `master` (synced with `develop`) so I guess you could do a rebase on `develop`.